src/qt/README.md

```diff
@@ -1,62 +1,61 @@
 ./src/qt/bitcoin-qt
 ```
 
-## Files and directories
+## Files and Directories
 
-### forms
+#### /forms
 
-Contains [Designer UI](https://doc.qt.io/qt-5.9/designer-using-a-ui-file.html) files. They are created with [Qt Creator](#using-qt-creator-as-ide), but can be edited using any text editor.
+- A directory which contains [Designer UI](https://doc.qt.io/qt-5.9/designer-using-a-ui-file.html) files. These files specify the characteristics of form elements in XML. Qt UI files are normally altered with [Qt Creator](#using-qt-creator-as-ide), but can be edited using any text editor.
 
-### locale
+#### /locale
 
-Contains translations. They are periodically updated. The process is described [here](/doc/translation_process.md).
+- Contains translations of the strings used throughout the GUI. They are periodically updated and an effort is made to support as many languages as possible. The process of contributing translations is described [here](/doc/translation_process.md).
 
-### res
+#### /res
 
-Resources such as the icon.
+ - Contains graphical resources used to enhance the UI experience.
 
-### test
+#### /test
 
-Tests.
+- Functional tests used to ensure proper functionality of the GUI. Significant changes to the GUI code normally require new tests to be written.
 
-### bitcoingui.(h/cpp)
+#### bitcoingui.(h/cpp)
 
-Represents the main window of the Bitcoin UI.
+- Represents the main window of the Bitcoin UI.
 
-### \*model.(h/cpp)
+#### \*model.(h/cpp)
 
-The model. When it has a corresponding controller, it generally inherits from  [QAbstractTableModel](https://doc.qt.io/qt-5/qabstracttablemodel.html). Models that are used by controllers as helpers inherit from other Qt classes like [QValidator](https://doc.qt.io/qt-5/qvalidator.html).
+- The model. When it has a corresponding controller, it generally inherits from  [QAbstractTableModel](https://doc.qt.io/qt-5/qabstracttablemodel.html). Models that are used by controllers as helpers inherit from other Qt classes like [QValidator](https://doc.qt.io/qt-5/qvalidator.html).
+- ClientModel is used by the main application `bitcoingui` and several models like `peertablemodel`.
 
-ClientModel is used by the main application `bitcoingui` and several models like `peertablemodel`.
+#### \*page.(h/cpp)
 
-### \*page.(h/cpp)
+- A controller. `:NAMEpage.cpp` generally includes `:NAMEmodel.h` and `forms/:NAME.page.ui` with a similar `:NAME`.
 
-A controller. `:NAMEpage.cpp` generally includes `:NAMEmodel.h` and `forms/:NAME.page.ui` with a similar `:NAME`.
+#### \*dialog.(h/cpp)
 
-### \*dialog.(h/cpp)
+- Various dialogs, e.g. to open a URL. Inherit from [QDialog](https://doc.qt.io/qt-5/qdialog.html).
 
-Various dialogs, e.g. to open a URL. Inherit from [QDialog](https://doc.qt.io/qt-5/qdialog.html).
+#### paymentserver.(h/cpp)
 
-### paymentserver.(h/cpp)
+- Used to process BIP21 payment URI requests. Also handles URI based application switching (e.g. when following a bitcoin:... link from a browser).
```

Comment:
Maybe add `(deprecated)`?

Reply:
Done in f6584d9c84f4d840dd5cb49f7dcc9984e9a16304